rosco-web/config/rosco.yml

```diff
@@ -5,7 +5,9 @@ rosco:
   configDir: /some/path/to/config/packer
   jobs:
     local:
-      timeoutMinutes: 20
+      timeoutMinutes: 30
```

Comment:
@robumberger can you ack that you tested not setting this param with Linux bake?  @duftler once rob acks, we are good with pull request 171, 1390 and 1146.